### PR TITLE
Fixing drawer design, after tab redesign

### PR DESCRIPTION
**Pull request by mimse**
Fixing drawer design after tab re-design

image

image

### CODE REVIEW

File: libs/dh/charges/feature-prices/src/lib/drawer/charge-content/details-header/dh-charge-details-header.component.html [hunk 1]
```diff
@@ -16,9 +16,9 @@
 -->
 
 <ng-container *ngIf="charge">
-  <div class="container" *transloco="let t; read: 'charges'">
+  <ng-container class="container" *transloco="let t; read: 'charges'">
```
Comment (mimse) [inline libs/dh/charges/feature-prices/src/lib/drawer/charge-content/details-header/dh-charge-details-header.component.html:19]:
```suggestion
  <ng-container *transloco="let t; read: 'charges'">
```

File: libs/ui-watt/src/lib/foundations/typography/_typography.import.scss [hunk 2]
```diff
@@ -62,7 +62,7 @@ $font-sizes: (
 }
 
 @mixin watt-headline-1 {
-  color: var(--watt-typography-headline-color);
+  color: var(--watt-typography-text-color);
```
Comment (dzhavat) [inline libs/ui-watt/src/lib/foundations/typography/_typography.import.scss:65]: question: why not change the color of `--watt-typography-headline-color` instead?

File: libs/ui-watt/src/lib/components/drawer/watt-drawer.component.scss [hunk 3]
```diff
@@ -58,13 +58,6 @@
     }
   }
 
-  &--normal,
-  &--large {
-    .watt-drawer__actions {
-      float: right;
-    }
-  }
```
Comment (dzhavat) [inline libs/ui-watt/src/lib/components/drawer/watt-drawer.component.scss:66]: praise: good that we got rid of the `float`

File: libs/ui-watt/src/lib/components/drawer/watt-drawer-actions-content.component.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  template: `<ng-content></ng-content>`,
+  styles: [
+    `
+      :host {
+        margin-left: var(--watt-space-ml);
+      }
+    `,
+  ],
+})
+export class WattDrawerActionsContentComponent {}
```
Comment (dzhavat) [inline libs/ui-watt/src/lib/components/drawer/watt-drawer-actions-content.component.ts:30]: suggestion: looks like the component is used primarily with a heading, so we can also call it `watt-drawer-heading`

File: libs/dh/admin/feature-user-management/src/lib/drawer/dh-user-drawer.component.html [hunk 5]
```diff
@@ -16,7 +16,14 @@
 -->
 <ng-container *transloco="let t; read: 'admin.userManagement.drawer'">
   <watt-drawer #drawer size="large" (closed)="onClose()">
-    <watt-drawer-topbar></watt-drawer-topbar>
+    <watt-drawer-topbar>
+      <dh-user-status *ngIf="selectedUser" [status]="selectedUser.status"></dh-user-status>
+    </watt-drawer-topbar>
+    <watt-drawer-actions-content>
+      <div class="user-name__grid">
```
Comment (dzhavat) [inline libs/dh/admin/feature-user-management/src/lib/drawer/dh-user-drawer.component.html:23]: suggestion: the extra element was necessary before, when there was a status badge under the heading, but the badge is not here anymore. The extra element and styling can be removed.

File: libs/dh/admin/feature-user-management/src/lib/drawer/roles/dh-role-drawer.component.html [hunk 6]
```diff
@@ -16,8 +16,20 @@
...
+        <dh-role-status [status]="userRoleWithPermissions.status"></dh-role-status>
+      </ng-container>
+    </watt-drawer-topbar>
+    <watt-drawer-actions-content *ngIf="drawer.isOpen">
+      <ng-container *rxLet="userRoleWithPermissions$ as userRoleWithPermissions">
+        <div class="role-name__grid">
+          <h2 class="role-name__headline">
+            {{ userRoleWithPermissions.name }}
+          </h2>
+        </div>
```
Comment (dzhavat) [inline libs/dh/admin/feature-user-management/src/lib/drawer/roles/dh-role-drawer.component.html:30]: suggestion: the extra element was necessary before, when there was a status badge under the heading, but the badge is not here anymore. The extra element and styling can be removed.

File: libs/ui-watt/src/lib/foundations/typography/_typography.import.scss (hunk 2 above)
Comment (mimse) [reply libs/ui-watt/src/lib/foundations/typography/_typography.import.scss:65]: That would be better

Comment (mimse) [reply libs/ui-watt/src/lib/foundations/typography/_typography.import.scss]: Resolved in https://github.com/Energinet-DataHub/greenforce-frontend/pull/1497/commits/5e8c97e4f562ff9bdd8db4384141da5da112bb4b

File: libs/ui-watt/src/lib/components/drawer/watt-drawer-actions-content.component.ts (hunk 4 above)
Comment (mimse) [reply libs/ui-watt/src/lib/components/drawer/watt-drawer-actions-content.component.ts]: Resolved in https://github.com/Energinet-DataHub/greenforce-frontend/pull/1497/commits/0c5088e6dfc35ef80e278709b0dce6a24774bf30

**Review by dzhavat — APPROVED** [review]

**Pull request merged**